### DISCUSSION TITLE
Add datum to sprite

### DISCUSSION THREAD

**Discussion by daxliniere** [Bodmer/TFT_eSPI#1512] [New processor/display support or capability request]
Hey Bodmer,
Would it be possible for you to implement a datum function to sprites, please? Same as datum for text.

Happy holidays,
Dax.

**Comment by Bodmer** [comment]
I am not sure exactly what you are asking for. Please explain further.

**Reply by daxliniere** [reply]
Hi Bodmer,
Thanks for your reply. Since I posted this, I did some more digging and found this is possible if you use pushRotated since you can specify a pivot point, which is the same as the datum idea I was thinking of.
I guess it would be useful if this feature was added to pushSprite as well.

Also, is there a list of all draw functions (pushRotated, fillRect, etc) online? I thought it might be in the wiki.

All the best!
Dax.
